tests/chainermn_tests/extensions_tests/test_checkpoint.py

```diff
@@ -49,12 +51,12 @@ def test_stats(self):
         stats = _CheckpointStats()
         assert isinstance(stats.report(), str)
 
-    def test_filename_converters(self):
+    def check_filename_converters(self, xp):
         checkpointer = create_multi_node_checkpointer(name='hoge',
                                                       comm=self.communicator,
                                                       cp_interval=23,
                                                       gc_interval=32)
-        nums = [np.random.randint(4096) for _ in range(234)]
+        nums = [int(xp.random.uniform() * 4096) for _ in range(234)]
```

Comment:
is this caused by the same problem with https://github.com/chainer/chainer/pull/8346/files#r346664255

Reply:
No, it is just because chainerx doesn't have `randint()` so instead we have to use `random.uniform()`, which returns floating-point values.